infra/gcp/clusters/modules/gke-cluster/variables.tf

```diff
@@ -51,3 +51,9 @@ variable "release_channel" {
   More information about release channels can be found here : https://cloud.google.com/kubernetes-engine/docs/concepts/release-channels.
 EOF
 }
+
+variable "enable_node_local_dns_cache" {
+  description = "If this cluster want to have NodeLocal DNSCache enabled"
```

Comment:
non-blocking nit: s/want/wants, though also I think it's more like "If this cluster should have NodeLocal DNSCache enabled", since the cluster wants for nothing 🙃

Reply:
lol, cluster wants nothing, so true!